CONTRIBUTING.md

```diff
@@ -1,31 +1,102 @@
-# How to become a contributor and submit your own code
+# Contributing to this Chef Cookbook
 
-## Contributor License Agreements
+## Submitting Issues
 
-We'd love to accept your sample apps and patches! Before we can take them, we
-have to jump a couple of legal hurdles.
+Not every contribution comes in the form of code. Submitting, confirming, and triaging issues is an important task for any project. We use GitHub to track all project issues.
 
-Please fill out either the individual or corporate Contributor License Agreement
-(CLA).
+If you are familiar with Chef and know the component that is causing you a problem, you can file an issue in the corresponding GitHub project.
```

Comment:
Typically we direct customer issues to Cloud Support, not directly via GitHub.

Let's remove this and/or sync offline with Harp to see how we want to phrase this.